<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window3</class>
 <widget class="QMainWindow" name="window3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>How many:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bank:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="window3_new_entry_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Entry</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="window3_how_many_field">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="window3_generate_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="window3_bank_field">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="window3_type_field">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>visa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mastercard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>amex</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>303</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
